requirements.txt

```diff
@@ -1,10 +1,10 @@
-kubernetes==9.0.1
-jupyterhub==1.1.0
-jupyterhub-kubespawner==0.12.0
+kubernetes==11.0.0
+jupyterhub==1.2.1
+git+https://github.com/mroman-redhat/oauthenticator.git@https-test
```

Comment:
Since this works, can you submit a PR to https://github.com/opendatahub-io/oauthenticator and when it is merged, update this to point to the odh-io/oauthenticator repo?